src/controllers/register.js

```diff
@@ -0,0 +1,25 @@
+function createUser(username, password) {
+  // TODO write user info to db and return result [bool]
+  console.log(username, password);
+  return true;
+}
+
+export const register = async (req, res) => {
+  let username = req.body.username;
+  let password = req.body.password;
+
+  let result = createUser(username, password);
+  if (result) {
+    res.send({
+      error_code: 0,
+      msg: 'ok',
+    });
+    return;
```

Comment:
If you want to send error code(status code), you can use `res.status(error_code)`.

I recommend that you use early return like above code.

I think void return is meaningless code.

```suggestion
    return  res.status(400).send({
      msg: 'ok',
    });
```